sklearn/linear_model/logistic.py

```diff
@@ -1213,7 +1213,7 @@ def fit(self, X, y, sample_weight=None):
             _dtype = np.float64
 
         X, y = check_X_y(X, y, accept_sparse='csr', dtype=_dtype,
-                         order="C")
+                         order="C",accept_large_sparse=True)
```

Comment:
Note that this [is solver dependent](https://github.com/scikit-learn/scikit-learn/issues/2969#issuecomment-309400363) : as far as I could tell newton-cg and lbfgs worked while sag failed..

Reply:
yes it is solver dependent, shouldn't this work this way only?

Reply:
Well maybe reflect that SAG doesn't accept those with ?
```py
accept_large_array=(self.solver not in ['sag', 'saga'])
```
 [need to check which ones work..]